FAQ.md

```diff
@@ -381,3 +380,11 @@ it can [cause
 problems](https://groups.google.com/d/msg/golang-nuts/AnMr9NL6dtc/UnyUUKcMCAAJ).
 If your dependers are using `dep`, this is not a concern, as `dep` takes care of
 stripping out nested `vendor` directories.
+
+## How do I configure a dependency that doesn't tag its releases?
+
+Add a constraint to your manifest that specifies `branch: "master"`
+for the dependency. `dep` will deduce the current revision number of your
```

Comment:
s/`dep`/`dep ensure`/ - just to make it clear what phase (e.g., not before `dep init`) in which this guidance applies.